File: aoc2015/day1/part1.go
package main 

import "fmt"
import "io/ioutil"
import "log"

func main(){

	content,err := ioutil.ReadFile("input.txt")

	if(err !=nil){
		log.Fatal(err)
	}
	currentFloor := 0
	for _,value := range string(content) {
		if value=='(' {
			currentFloor+=1
		} else if value ==')' {
			currentFloor -=1
		} 
	}
	fmt.Println(currentFloor)
	
}

File: aoc2015/day1/part1.py

def main():
    with open('input.txt','r') as f:
        p=f.read()

    currentFloor = 0 
    for i in p:
        if i=='(':
            currentFloor+=1
        elif i==')':
            currentFloor-=1
    print(currentFloor)


main()
